README.md

```diff
@@ -122,16 +139,19 @@ Run the script `python app/database_etl/etl_main.py`.
 Confirm that the data has indeed been migrated by checking pgAdmin 4.
 
 ## Generating Airtable Fields Config from SOP
```

Comment:
@simonarocco this is no longer relevant right? should we delete

Reply:
ya @annoyingpr you can delete this section about generating airtable fields config from SOP